fix: function -> class method

TheGoldKay opened the pull request.

TheGoldKay committed:
fix: function -> class method

novafacing commented:
This doesn't seem correct. `keypressed` shouldn't be a class method?

novafacing commented:
Ah, nvm. I see the intent. Since this is just supposed to be sample code, however. I don't think it matters as is.

TheGoldKay commented:
> This doesn't seem correct. `keypressed` shouldn't be a class method?

That's the fix, the old code used the -> rather than =>, and thus the first argument was the class itself, and therefore rather than passing a key it was passing the game object itself.



> Ah, nvm. I see the intent. Since this is just supposed to be sample code, however. I don't think it matters as is.

I'm well aware, that's why I thought you were gonna approve something that literally means nothing to you and at the same time means a lot to me, but whatever.

novafacing commented:
Ok, thanks for the explanation. In the future, a message on your PRs will help maintainers understand why it's being made. I'll test when I have a minute.

TheGoldKay commented:
> Ok, thanks for the explanation. In the future, a message on your PRs will help maintainers understand why it's being made. I'll test when I have a minute.

I understand, a PR should indeed have a brief explanation of the issue being addressed and fixed.

TheGoldKay committed:
workflow update: github actions (deprecated) artifact: v3 -> v4

TheGoldKay committed:
branch update for  nhartland/love-build: 'master' -> 'main'

TheGoldKay commented:
@novafacing I also fixed GitHub actions workflow, updating artifact (**_v3_** to **_v4_**), and changed **nhartland/love-build** default branch (**_master_** to **_main_**).

novafacing commented:
Thanks for your contribution!
